Gems/EMotionFX/Code/Source/Editor/ColliderHelpers.cpp

```diff
@@ -292,9 +390,48 @@ namespace EMotionFX
             CommandColliderHelpers::RemoveCollider(actor->GetID(), joint->GetNameString(), type, shapeIndex, &commandGroup);
         }
 
-        CommandColliderHelpers::AddCollider(actor->GetID(), joint->GetNameString(), type, AZStd::string(clipboardContents.data(), clipboardContents.size()), shapeIndex, &commandGroup);
+        CommandColliderHelpers::AddCollider(
+            actor->GetID(),
+            joint->GetNameString(),
+            type,
+            AZStd::string(clipboardContents.data(), clipboardContents.size()),
+            shapeIndex,
+            &commandGroup);
 
         AZStd::string result;
         CommandSystem::GetCommandManager()->ExecuteCommandGroup(commandGroup, result);
     }
+
+    bool ColliderHelpers::IsInRagdoll(const QModelIndex& modelIndex)
```

Comment:
Why the inconsistency of the naming? 
ColliderHelpers::IsInRagdoll
ColliderHelpers::NodeHasClothCollider
ColliderHelpers::NodeHasHitDetectio